src/instructlab/model/evaluate.py

```diff
@@ -333,6 +333,11 @@ def launch_server(
                     fg="yellow",
                 )
             eval_serve.vllm.vllm_args.extend([tps_prefix, str(gpus)])
+        else:
```

Comment:
I don't think this is quite right.  I think you need to check whether gpus or effective_gpus > 0.  effective gpus being > 0 means the value for gpus is coming from serve config.

Reply:
So the `if gpus:` statement above this would cover any case where `gpus` is greater than 0 -  so here I think I can just check `elif effective_gpus < 1`?

Reply:
Yes!